diff --git a/node_modules/@remix-run/dev/compiler.js b/node_modules/@remix-run/dev/compiler.js
index eaa9bce..3e6a373 100644
--- a/node_modules/@remix-run/dev/compiler.js
+++ b/node_modules/@remix-run/dev/compiler.js
@@ -248,7 +248,7 @@ async function createServerBuild(config, options) {
       resolveDir: config.serverBuildDirectory
     },
     outfile: path.resolve(config.serverBuildDirectory, "index.js"),
-    platform: "node",
+    platform: process.env.SERVER_PLATFORM || "node",
     format: "cjs",
     target: options.target,
     inject: [reactShim],
